### ISSUE TITLE
HTML5 Client does not send resize event when closing chat or user list

### ISSUE THREAD

**Issue by lfzawacki** [bigbluebutton/bigbluebutton#4935]
This event is used by the video-dock component to re-calculate the size of the video windows and thus when it's not sent the videos will remain smaller than they should be when the user list or chat is closed.

This video shows the bug: https://www.youtube.com/watch?v=9FFxaZaSF1U

First the web cam is shared, then user list and chat are opened and the video resizes. When the userlist and chat are closed the web cam remains small, but when the window is resized (when the event is fired) it takes up it's normal size.

**Comment by OZhurbenko** [comment]
Hey @lfzawacki 
Actually it does send the "resize" event. Both on opening and closing. The whiteboard listens to it, and it's very easy to test.
You can check if the presentation scales when user list/chat/closed-captions/polling/presenter-controls push it or give it more space.

The event is definitely dispatched, that's not an issue.

**Comment by OZhurbenko** [comment]
The other thing is that the function is called [injectWbResizeEvent](https://github.com/bigbluebutton/bigbluebutton/blob/master/bigbluebutton-html5/imports/ui/components/presentation/resize-wrapper/component.jsx) , since it was used for the whiteboard only.
Maybe it's a good idea to rename it then.

**Comment by OZhurbenko** [comment]
Ah, I got it.
Lucas, try to wrap the call to adjustVideos into setTimeout:

```
  adjustVideos() {
    setTimeout(window.adjustVideos('webcamArea', true), 0);
  }
```

That should fix it.

**Comment by lfzawacki** [comment]
Yeah, I tested it and the event is actually sent but when it arrives the container is still the old size and the resize does not work. I'm going to solve it by making the resize code wait one tick before executing.

**Comment by antobinary** [comment]
Addressed in #4955
